src/client/js/Panels/MetaEditor/MetaEditorControl.DiagramDesignerWidgetEventHandlers.js

```diff
@@ -120,6 +121,20 @@ define(['js/logger',
                     }
                 }
                 return;
+            } else if (self._connType === MetaRelations.META_RELATIONS.MIXIN) {
+                node = self._client.getNode(sourceId);
+                if (node) {
+                    mixinCheckResult = node.canSetAsMixin(targetId);
+
+                    if (mixinCheckResult.isOk === false) {
+                        dialog.alert('Invalid mixin target',
```

Comment:
Add an error or warning logging somewhere here, if we open the console we will get a line number and or stack trace.